src/main/java/nextstep/auth/AuthConfig.java

```diff
@@ -1,36 +1,33 @@
 package nextstep.auth;
 
-import nextstep.auth.authentication.BasicAuthenticationFilter;
-import nextstep.auth.authentication.BearerTokenAuthenticationFilter;
-import nextstep.auth.authentication.UsernamePasswordAuthenticationFilter;
+import lombok.RequiredArgsConstructor;
+import nextstep.auth.authentication.UsernamePasswordAuthFilter;
 import nextstep.auth.authorization.AuthenticationPrincipalArgumentResolver;
 import nextstep.auth.context.SecurityContextPersistenceFilter;
+import nextstep.auth.authentication.BasicAuthFilter;
+import nextstep.auth.authentication.BearerTokenAuthFilter;
+import nextstep.auth.user.UserDetailService;
+import nextstep.auth.token.TokenAuthInterceptor;
 import nextstep.auth.token.JwtTokenProvider;
-import nextstep.auth.token.TokenAuthenticationInterceptor;
-import nextstep.member.application.LoginMemberService;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 import java.util.List;
 
+@RequiredArgsConstructor
 @Configuration
 public class AuthConfig implements WebMvcConfigurer {
-    private LoginMemberService loginMemberService;
-    private JwtTokenProvider jwtTokenProvider;
-
-    public AuthConfig(LoginMemberService loginMemberService, JwtTokenProvider jwtTokenProvider) {
-        this.loginMemberService = loginMemberService;
-        this.jwtTokenProvider = jwtTokenProvider;
-    }
+    private final UserDetailService loginMemberService;
```

Comment:
변수명도 변경되면 좋을 것 같아요 😃 
```suggestion
    private final UserDetailService userDetailService;
```